src/lib/types/index.ts

```diff
@@ -1,35 +1,38 @@
 // Create a new types file to centralize interfaces
 export interface Process {
-  pid: number;
-  ppid: number;
-  name: string;
-  cpu_usage: number;
-  memory_usage: number;
-  status: string;
-  user: string;
-  command: string;
-  threads?: number;
+  pid: number
```

Comment:
I usually only wanna see code changes that are directly related to the purpose of the PR, can we revert here?

Reply:
yeah Done